data/lib/core/tile.lua

```diff
@@ -32,3 +32,44 @@ function Tile.relocateTo(self, toPosition)
 
 	return true
 end
+
+function Tile.isWalkable(self, check)
+  local ground = self:getGround()
```

Comment:
This is the downside of OTC imo, ppl might start out with scripting on OTC and then think we use spaces, but we should try to keep a consistency in our code style, that is why my project uses tabs xD

So to my review, use tabs insted of spaces, most of the OT stuff uses tabs.
Most coding is done via spaces, since you can modify your editor to show a space as 2 spaces (while you then press tab the next editor might show it as 5 spaces insted of forcing people to use 2 spaces).

If people wanna argue about using parentheses around if statment etc that is a diffrent story tho :P